### DISCUSSION TITLE
'Homepage' images path ?

### DISCUSSION THREAD

**Discussion by gladiatormax** [tteck/Proxmox#3230] [Help Needed] [answered]
I am trying to update the favicon.png on homepage dashboard.
I uploaded the files to /opt/homepage/public/images/*.*
In my .Yaml i point "favicon: /images/favicon.ico" but with no luck.
What is the relative path for images, i already tried all combinations but with no luck.
Also restarted the container.

**Comment by tesla-76** [comment, marked as answer]
The path is:   **_`/opt/homepage/public/`_** as for normal icons, but in update I encountered problems.
I solved by putting in yaml file:
`favicon: https://your-home-page-web-address.net/yourfavicon.png`

Attention, to make the images viewable you have to restart the LXC container once you have uploaded its to the **_/public_** directory.

**Reply by gladiatormax** [reply]
Thank you, it is working with this workaround.
It was the same for the background, hopefully they will fix it in a next update.
